db/migrate/20221018175100_add_user_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: 'author_id', index: true
```

Comment:
- [ ] Kindly point to the author instead of user, based on the requirements.

Reply:
Done